fe/fe-core/src/main/java/com/starrocks/connector/RemotePathKey.java

```diff
@@ -14,46 +14,20 @@
 
 package com.starrocks.connector;
 
-import org.apache.hudi.common.table.timeline.HoodieInstant;
-import org.apache.hudi.common.table.timeline.HoodieTimeline;
-import org.apache.hudi.common.table.view.HoodieTableFileSystemView;
-
 import java.util.Objects;
-import java.util.Optional;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.locks.ReentrantLock;
 
 public class RemotePathKey {
     private final String path;
     private final boolean isRecursive;
-
-    // The table location must exist in HudiTable
-    private final Optional<String> hudiTableLocation;
-
-    public static class HudiContext {
-        // ---- concurrent initialization -----
-        public AtomicBoolean init = new AtomicBoolean(false);
-        public ReentrantLock lock = new ReentrantLock();
-        // ---- actual fields -----
-        public HoodieTableFileSystemView fsView = null;
-        public HoodieTimeline timeline = null;
-        public HoodieInstant lastInstant = null;
-    }
-
-    private HudiContext hudiContext;
+    private RemoteFileLoadingContext loadingContext;
```

Comment:
Better remove this from RemotePathKey, RemoteFileLoadingContext is not part of key

Reply:
No, it has to bind to this remote path key.  

The case is, then you load files based on some remote path keys, some fields are shared.  So how do you share those field? The only parameter is remote path key.